### DISCUSSION TITLE
v3.0-beta

### DISCUSSION THREAD

**Discussion by tomakino** [tomakino/CSLyric#36] [Announcements]
- 适配Android15
- 适配留声机
- 适配ColorOS流体云
- 更好的动画系统
- 更好的界面

<hr /><em>This discussion was created from the release <a href='https://github.com/tomakino/CSLyric/releases/tag/v3.0-beta'>v3.0-beta</a>.</em>

**Comment by aboboge** [comment]
Color14怎么更新后，显示时间时时间没法隐藏了😰

**Reply by 3085300994** [reply]
同问，一样的情况，时间无法隐藏，歌词在时间和流体云之间，占很小部分![Screenshot_2024-11-26-10-49-59-95_320a9a695de7cdce83ed5281148d6f19.png](https://github.com/user-attachments/assets/075c98ff-f91f-4175-86c3-d2c2198558b3)

**Reply by tomakino** [reply]
> Color14怎么更新后，显示时间时时间没法隐藏了😰

新版对流体云做了适配，可单独设置歌词在流体云处于显示中的宽度。

在播放时隐藏时间可以在视图控制器找到`clock`设置为在播放时隐藏

**Comment by 3085300994** [comment]
时间无法隐藏，歌词在时间和流体云之间，占很小部分

**Reply by tomakino** [reply]
新版对流体云做了适配，可单独设置歌词在流体云处于显示中的宽度。
在播放时隐藏时间可以在视图控制器找到`clock`设置为在播放时隐藏

**Comment by tomakino** [comment]
新版对流体云做了适配，可单独设置歌词在流体云处于显示中的宽度。
在播放时隐藏时间可以在视图控制器找到`clock`设置为在播放时隐藏

**Comment by 3085300994** [comment]
这是来自QQ邮箱的自动回复邮件。您发给我的邮件已经收到！幸苦了哦

**Comment by colochini-86** [comment]
最新版本哪里下载

**Comment by 3085300994** [comment]
这是来自QQ邮箱的自动回复邮件。您发给我的邮件已经收到！幸苦了哦
